Core/src/org/sleuthkit/autopsy/centralrepository/optionspanel/ShowCasesDialog.java

```
@@ -0,0 +1,189 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.centralrepository.optionspanel;
+
+import java.util.List;
+import javax.swing.JDialog;
+import javax.swing.JFrame;
+import org.openide.util.Exceptions;
+import org.openide.util.NbBundle.Messages;
+import org.openide.windows.WindowManager;
+import org.sleuthkit.autopsy.centralrepository.datamodel.CorrelationCase;
+import org.sleuthkit.autopsy.centralrepository.datamodel.EamDb;
+import org.sleuthkit.autopsy.centralrepository.datamodel.EamDbException;
+
+/**
+ * Dialog to display table of CorrelationCase information from the CR tab of options.
+ */
+@SuppressWarnings("PMD.SingularField") // UI widgets cause lots of false positives
+final class ShowCasesDialog extends JDialog {
+
+    private static final long serialVersionUID = 1L;
+
+    private final ShowCasesTableModel tableModel;
+    @Messages({"ShowCasesDialog.title_text=All Cases Details"})
+    /**
+     * Creates new form ShowCases Panel
+     */
+    ShowCasesDialog() {
+        super((JFrame) WindowManager.getDefault().getMainWindow(),
+                Bundle.ShowCasesDialog_title_text(),
+                true); // NON-NLS
+        tableModel = new ShowCasesTableModel();
+        initComponents();
+        try {
+            EamDb dbManager = EamDb.getInstance();
+            List<CorrelationCase> eamCases = dbManager.getCases();          
+            for(CorrelationCase eamCase : eamCases) {
+                tableModel.addEamCase(eamCase);
+            }
+        } catch (EamDbException ex) {
+            Exceptions.printStackTrace(ex);
```

Please log this or let the method throw the exception.